### ISSUE TITLE
Vscode not sending stats anymore.

### ISSUE THREAD

**Issue by yinx** [swdotcom/swdc-vscode#17]
I just noticed that after updating to the latest version (0.8.5) The vscode plugin doesn't seem to send the data to the app anymore.

When I try to log in again by using the log in command from the plugin, it logs me in just fine but still doesn't update the stats.

I'm on mojave 10.14.3 an vscode 1.31.1.

Did I miss something after the update or did a bug sneak in somewhere?

**Comment by andrewhli** [comment]
Hi there. Thanks for letting us know. While we're trying to recreate the bug from our side, could you reset the data connection by doing the following:

1. Navigate to your root directory: /Users/[your-username]
2. Delete the .software folder (it's a hidden folder, so you may have to adjust preferences to see it)
3. On VS Code, log into Code Time again
4. Type, and wait a minute for the stats to update (status bar, Code Time Dashboard)

Could you let us know if your stats update after doing these steps? Still trying to reproduce it our end.

**Comment by yinx** [comment]
Thanks for the quick reply.
I think that might just work. I just tried installing it on another mac and the connection works, so I suspect that will do it. Maybe something during the update got messed up or something..
I'll try it when I get back into the office tomorrow.
Thanks for looking into this.

**Comment by frohikey** [comment]
```VSCode: 1.32.1 / OS: Windows 10 / Code Time: 0.9.7```

I've got the same problem. No telemetry is being sent. I've tried to delete user folder (.software) and re-login. Only ```session.json``` has been recreated.  JWT token is set and epoch (vscode_lastUpdateTime) looks correct. Before deleting I saw another ```json``` with statistics but this one is still missing.

Text version of dashboard can be generated but basically I'm stuck with the same stats. And trust me I've been working like a crazy 🐒 today.

**Comment by andrewhli** [comment]
Thanks for letting us know! We introduced a new feature last week that allows one user to have multiple Code Time accounts on one device (enabled a Log Out option). Right now, it looks like those features are creating problems (always showing 0 min coding for some users). We are fixing this today/tomorrow by going back to a simpler version without those extra features. I'll be sure to let you know when that's fixed.

**Comment by frohikey** [comment]
Well, now I'm stuck with 6.5 hours logged everyday at start of my work. My Monday is displayed two times at the web dashboard. And today looks kinda familiar 😄 

![image](https://user-images.githubusercontent.com/13802274/54193859-3b8a4000-44bb-11e9-9ac5-d8ae3efe2377.png)

Is it possible to wipe out all data from account somehow? I believe that there's only an option for deactivating my account that only stops receiving telemetry.

**Comment by andrewhli** [comment]
Not to make light of these bugs, but I do appreciate your good humor. We're rolling out changes today/tomorrow that we believe will make accounts "unstuck". We'll then fix the bug that's duplicating the datagrams.

**Comment by frohikey** [comment]
Well I love extensions which are not meant to be "must-have" for my work. But they give magic sparkles to my life in IDEs. So good job guys!

I'm not sure that rolling out a new version gonna fix my historic data. I've been using this extension only for a couple of days. So I'm 100% ok with wiping them out. Is it possible somehow atm? Or I need to use another account?

**Comment by brettmstevens7** [comment]
@frohikey can you check if you see your current or stale data in your VS Code dashboard now? Is it both your dashboard and your status bar that are not updating, or just one of the two?

We'll fix the issue with the duplicate items in the feed - thanks for pointing that out. 

If you scroll down in your feed, you'll see previous days. The three circles chart is a daily summary that appears at the end of every day, at 11:59pm local time.

**Comment by oJshua** [comment]
Encountering this issue with `VSCode: 1.31.1 / OS: Debian Stretch (over ChromeOS Crostini) / Code Time: 0.10.0` since 3/7 (last data report).

Removed .software. Able to log in (sends OAuth to external Chrome instance?) but JWT doesn't get populated. This may be separate issue territory now.

**Comment by frohikey** [comment]
@brettmstevens7 I've always had both values in sync (dashboard + status bar) with 0 or other constant non-zero value. Yesterday my 6 hours constant value was deleted at the end of the day and only next 2 hours was logged in. But since than I think everything is working properly. Today code time started at 0, now I have +/- 1 hour which is correct.

TLDR: It seems it is fixed now for me. Thanks.

**Comment by frohikey** [comment]
Ah, unfortunately it doesn't work again. It's really random.
```
CODE TIME

No data available
```

![image](https://user-images.githubusercontent.com/13802274/54287177-72d51b80-45a5-11e9-9d01-d5135b4e028a.png)

It's extremely slow. If I click on the status bar, options will be displayed in 10sec+. Opening in-line dashboard took 1 minute with ```No data available```.

Well, and now status line is back. It's extremely all random. Sorry, I know very well how such bug reports suck.

![image](https://user-images.githubusercontent.com/13802274/54287956-f0e5f200-45a6-11e9-8f4a-e7acd5dfec03.png)

After 5 mins... it's 0min again :(

**Comment by jirmonder** [comment]
After updating VSCode it won't even show the status bar, open dashboard or anything, this is on Win10, VSCode V1.32.2.

Any tips ? keep the good work guys.

**Comment by andrewhli** [comment]
Hi everyone. Appreciate your patience. Today, we pushed Code Time v0.11.1 for VS Code, which we've tested for Win10 and MacOS. Here are the steps that we believe will fix the problems you brought up, but please let us know if you continue to see issues, and we'll fix from there:

1. Quit/exit VS Code

2. In your home directory, delete the .software folder (this is a hidden folder, and deleting it will reset the connection)
![image](https://user-images.githubusercontent.com/5068143/54399921-ca46ba80-467d-11e9-8844-ea92d79d1c4a.png)

3. Restart VS Code and check that Code Time is the latest version (v.0.11.1)
![image](https://user-images.githubusercontent.com/5068143/54399926-d3378c00-467d-11e9-9710-1de043672e60.png)

**Comment by prydda** [comment]
Just tried the above, it didnt reset my code time, still showing same numbers.

**Comment by frohikey** [comment]
Same here. I'm stuck with the constant number (3.2 hours) since morning.

**Comment by brettmstevens7** [comment]
Hi @frohikey - are you seeing correct data in your VS Code dashboard now? We made a change on Sunday. We're still working on the duplicate items appearing in your web feed now.
